src/parser/template/output.ts

```diff
@@ -14,8 +16,46 @@ export function parseOutput(x: schema.Output): TemplateOutput {
     value: parseExpression(x.Value),
     conditionName: ifField(x, 'Condition', assertString),
     description: ifField(x, 'Description', assertString),
-    exportName: ifField(x, 'Export', (e) =>
-      parseExpression(assertField(e, 'Name'))
-    ),
+    exportName: ifField(x, 'Export', (e) => ifField(e, 'Name', assertString)),
   };
 }
+
+export interface DeCdkCfnOutputProps {
+  value: any;
+  description?: string;
+  exportName?: string;
+  condition?: string;
+}
+
+export class DeCDKCfnOutput extends CfnElement {
+  private readonly value: any;
+  private readonly description?: string;
+  private readonly exportName?: string;
+  private readonly condition?: string;
```

Comment:
> Type of Condition is CfnCondition while in decdk, we can treat it as a string.
I guess that's a compilcation because we haven't done Conidtions yet in deCDK. Once everything is implemented we could probably get the `CfnCondition` from the evaluation context. This is good for now.

Reply:
Good point! I'll create an issue for this.